frontend/src/index.tsx

```diff
@@ -1,17 +1,32 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './App';
+import Post from './component/Post';
 import './index.css';
-import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <div className='Center'>
```

Comment:
It is common to have an `App` component. In `index.tsx` is usually all that stuff like the strict mode and required wrapper components for things like Redux and in these wrappers one adds the `App` component. Then in the `App` component there is the actual root component from a layout point of view.